src/misc/lv_text.h

```diff
@@ -75,39 +91,33 @@ typedef enum {
  * @param size_res pointer to a 'point_t' variable to store the result
  * @param text pointer to a text
  * @param font pointer to font of the text
- * @param letter_space letter space of the text
- * @param line_space line space of the text
- * @param max_width max width of the text (break the lines to fit this size). Set COORD_MAX to avoid
- * @param flag settings for the text from ::lv_text_flag_t
-
- * line breaks
+ * @param attributes the text attributes, flags for line break behaviour, spacing etc
  */
-void lv_text_get_size(lv_point_t * size_res, const char * text, const lv_font_t * font, int32_t letter_space,
-                      int32_t line_space, int32_t max_width, lv_text_flag_t flag);
+void lv_text_get_size(lv_point_t * size_res, const char * text, const lv_font_t * font,
+                      lv_text_attributes_t * attributes);
 
 /**
- * Give the length of a text with a given font
+ * Give the length of a text with a given font - kept for backwards compatibity with v8
  * @param txt a '\0' terminate string
  * @param length length of 'txt' in byte count and not characters (Á is 1 character but 2 bytes in
  * UTF-8)
  * @param font pointer to a font
  * @param letter_space letter space
  * @return length of a char_num long text
  */
-int32_t lv_text_get_width(const char * txt, uint32_t length, const lv_font_t * font, int32_t letter_space);
+int32_t lv_text_get_width_old(const char * txt, uint32_t length, const lv_font_t * font, int32_t letter_space);
```

Comment:
Do we really need this `_old` function? IMO we can remove it.

Reply:
We don't need to support v8 compatibility in v9.4. An these function will be priate anyways.